.github/workflows/rust.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         rust: [nightly]
         make:
           - name: clippy
-            task: "check"
+            task: "clippy"
```

Comment:
Good catch I created a CI make command Abd forgot to edit it. Can we change to that

Reply:
https://github.com/fleek-network/ursa/pull/142/commits/6a22c1d55330fcca3f1bb0abe6ae6756195128d9